cmd/controller/app/options/options.go

```diff
@@ -183,6 +191,8 @@ func (s *ControllerOptions) AddFlags(fs *pflag.FlagSet) {
 		"will be attempted.")
 	fs.StringVar(&s.Kubeconfig, "kubeconfig", defaultKubeconfig, ""+
 		"Paths to a kubeconfig. Only required if out-of-cluster.")
+	fs.Float32Var(&s.KubernetesAPIQPS, "kubernetes-api-qps", defaultKubernetesAPIQPS, "indicates the maximum QPS requests to the Kubernetes master")
+	fs.IntVar(&s.KubernetesAPIBurst, "kubernetes-api-burst", defaultKubernetesAPIBurst, "maximum burst for throttle requests to the Kubernetes master")
```

Comment:
`the maximum burst queries-per-second of requests sent to the Kubernetes apiserver` - perhaps with a link to somewhere with more details on how burst behaves, and how this should be set relative to the QPS?

What happens if I set this value *lower* than QPS?

Reply:
Added a validation rule

If you have a link very welcome! Been looking for any docs but the code comments is the furthest i got with useful info :(